### PR TITLE
Fix/disable save

### LINKED ISSUE #180: Disable Save-button clicks and and display a progress indicator while events are being written to API
In the UI, there's a possible lag between when a user selects Julkaise tapahtuma(t) and when the Julkaistun tapahtuman tallennus onnistui!-page is displayed. This lag can last several seconds.

During that time the Save-button should be disabled and in its place a circular work in progress-indicator gif should be displayed. Add animation for button shape transition if necessary.

### PR DESCRIPTION
**Pull request by jonraem**
Disable save button and display loader when data is being sent from Editor. Disable automatic sorting of events.

Fixes #180 and #192.

### CODE REVIEW

File: src/views/Editor/index.js [hunk 1]
```diff
@@ -118,10 +119,11 @@ var EditorPage = React.createClass({
         } else {
             return (
                 <span>
+                    <Loader loaded={!this.props.editor.isSending} scale={1}/>
                     <RaisedButton
                         style={buttonStyle}
                         primary={true}
-                        disabled={this.props.user && !this.props.user.organization}
+                        disabled={this.props.editor.isSending || (this.props.user && !this.props.user.organization)}
                         label={labelText}
```
Comment (Rikuoja) [inline src/views/Editor/index.js]: Trying this out, in some cases it really takes a long while to publish an event. Should we change the button text from labelText ("julkaise tapahtuma") to publishing ("julkaistaan tapahtumaa...") to make the user understand why they have to wait and the button is disabled?

Comment (aleksisalonen) [inline src/views/Editor/index.js]: Isn't the button showing a loader? Or is the loader icon diplayed separately?

If the latter, then I suppose we should replace the original text witht "Julkaistaan tapahtumaa...".

(I thought that the publishing times had come down to reasonable lengths.)

**Comment by aleksisalonen** [comment]
BTW, this issue is a duplicate of #180.

File: src/components/FormFields/index.js [hunk 2]
```diff
@@ -139,7 +139,7 @@ class FormFields extends React.Component {
                 )
             }
         }
-        newEvents = sortBy(newEvents, (events) => (events.props.event.start_time))
+        // newEvents = sortBy(newEvents, (events) => (events.props.event.start_time))
```
Comment (Rikuoja) [inline src/components/FormFields/index.js]: This fix has the (unintended) consequence that also the events returned by the recurring modal dialog remain unsorted. We should probably sort them when returning from the modal, but not sort any events added by hand.

**Comment by jonraem** [comment]
Sorting fixed.

**Pull request merged**